src/OSPSuite.Core/Domain/Services/QuantityValuesUpdater.cs

```diff
@@ -126,7 +126,7 @@ private IParameter getParameter(ModelConfiguration modelConfiguration, PathAndVa
          }
 
          //If the value is defined, this will be used instead of the formula (even if set previously)
-         if (pathAndValueEntity.Value != null)
+         if (pathAndValueEntity.Value.HasValue && !double.IsNaN(pathAndValueEntity.Value.Value))
```

Comment:
this looks like a method to specicy if a double? is defined (we probably have this ?)
Also this needs a test IMO

Reply:
Yeah, I'll see about a test. This doesn't have a specific tester except through integration because it's internal.